# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/samba-fstab.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./client/umount.cifs.c.orig	2005-07-28 15:19:53.000000000 +0200
+++ ./client/umount.cifs.c	2006-01-21 17:28:30.000000000 +0100
@@ -34,7 +34,6 @@
 #include <errno.h>
 #include <string.h>
 #include <mntent.h>
-#include <fstab.h>
 
 #define UNMOUNT_CIFS_VERSION_MAJOR "0"
 #define UNMOUNT_CIFS_VERSION_MINOR "5"
